.editorconfig

```diff
@@ -0,0 +1,18 @@
+# http://editorconfig.org
```

Comment:
Super handy formatting rules to use with https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

Specifically will automatically trim trailing whitespace and add newlines at bottom of files.